modules/supported-platforms-for-managed-clusters.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.adoc
+
+:_content-type: PROCEDURE
+[id="supported-platforms-for-managed-clusters_{context}"]
+= supported platforms for managed clusters
+
+You can manually specify the platform type for managed clusters if you do not want to set the platform type automatically.
+
+The {ai-full} supports the following platform types:
+* `BareMetal`
+* `VSphere`
+* `None`
+
+.Procedure
+
+* Select the platform in the `AgentClusterInstall` custom resource (CR) by changing the value of the `spec.platformType` parameter:
++
+[source,yaml]
+----
+apiVersion: extensions.hive.openshift.io/v1beta1
+kind: AgentClusterInstall
+metadata:
+  name: cluster
+  namespace: cluster
+spec:
+  platformType: VSphere
+----
+
+In some cases, the system automatically sets the platform type:
```

Comment:
It doesn't seem like this list is rendering correctly in the doc preview

Reply:
Same issue here as above I think, try adding a blank line

Reply:
Updated as suggested.